app/views/spree/admin/subscription_orders/index.html.erb

```diff
@@ -0,0 +1,96 @@
+
+<% content_for :table_filter_title do %>
+  <%= t('spree.filter') %>
+<% end %>
+
+<% content_for :table_filter do %>
+  <div data-hook="admin_orders_index_search">
+    <%= search_form_for [:admin, @search], url: admin_subscription_subscription_orders_path(@subscription) do |f| %>
+      <div class="row">
+        <div class="field-block col-12 col-md-6 col-lg-7">
+          <div class="date-range-filter field">
+            <%= label_tag :q_created_at_gt, t('spree.date_range') %>
+            <div class="date-range-fields input-group">
+              <%= f.text_field :created_at_gt, class: 'datepicker form-control datepicker-from', placeholder: t('spree.start') %>
+
+              <div class="input-group-prepend input-group-append">
+                <span class="input-group-text range-divider">
+                  <i class="fa fa-arrow-right"></i>
+                </span>
+              </div>
+
+              <%= f.text_field :created_at_lt, class: 'datepicker form-control datepicker-to', placeholder: t('spree.stop') %>
+            </div>
+          </div>
+        </div>
+        <div class="field-block col-12 col-md-6 col-lg-5">
+          <div class="field">
+            <%= label_tag :q_email_start, t('spree.email') %>
+            <%= f.text_field :email_start %>
+          </div>
+        </div>
+      </div>
+
+      <div class="clearfix"></div>
+
+      <div class="actions filter-actions">
+        <div data-hook="admin_orders_index_search_buttons">
+          <%= button_tag t('spree.filter_results'), class: 'btn btn-primary' %>
+        </div>
+      </div>
+    <% end %>
+  </div>
+<% end %>
+
+<fieldset class="no-border-bottom">
+  <legend><%= t('spree.admin.subscription_orders.index.title') %></legend>
+
+  <%= paginate @subscription_orders, theme: 'solidus_admin' %>
+
+  <% if @subscription_orders.any? %>
+    <table id="listing_subscription_orders" class="index">
+      <thead>
+        <tr data-hook="admin_subscription_orders_index_headers">
+          <th><%= sort_link @search, :created_at %></th>
+          <th><%= sort_link @search, :number %></th>
+          <th><%= sort_link @search, :email %></th>
+          <th class="align-right"><%= sort_link @search, :total %></th>
+          <th data-hook="admin_orders_index_header_actions" class="actions"></th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @subscription_orders.each do |order| %>
+          <tr class="admin_subscription_order">
+            <td><%= l order.created_at.to_date %></td>
+            <td><%= link_to order.number, edit_admin_order_path(order) %></td>
+            <td>
+              <% if order.user %>
+                <%= link_to order.email, edit_admin_user_path(order.user) %>
+              <% else %>
+                <%= order.email %>
+              <% end %>
+              <%= link_to_with_icon('email', t('spree.actions.send_email'), "mailto:#{order.email}", no_text: true) %>
+            </td>
```

Comment:
Given that the user is always going to be the same, perhaps we can remove this column?

Reply:
Instead of the user, it would be useful to display the order's state if possible.

Reply:
Sure: https://github.com/solidusio-contrib/solidus_subscriptions/pull/192/commits/5cc4b5d32f50c591598ece6dbd81a2455e28ada2